aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -338,6 +338,12 @@ void resize_tensor(Tensor* output) {
                                      name:nil];
 }
 
+MPSGraphTensor* mpsGraphScalarPlaceHolder(MPSGraph *mpsGraph, MPSDataType dataType) {
```

Comment:
Hi @qqaatw , it will be great if we can combine this function with the above [one](https://github.com/pytorch/pytorch/pull/88555/files#diff-c35f3c3f2f56d77db0e1b290ea4ca49e78e016c5732d319863ebb1c88cd775bcR335) and use it. You can extend that to take in the MPSDataType and default to the ones passed with Scalar Tensor.

Reply:
Hi @kulinseth, it seems that the link doesn't point me to a specific line. Do you mean that we reuse this function? `MPSGraphTensor* mpsGraphScalarPlaceHolder(MPSGraph *mpsGraph, const Scalar& scalar)`. 

In this scenario I think we should specify `ScalarType::Float` instead of using the dtypes of `beta` or `threshold` here as they're previously casted to float32 to make the graph's dtypes consistent. As a result, adding an overloading of `mpsGraphScalarPlaceHolder` seems to be a concise option like what we do for `mpsGraphRankedPlaceHolder`.